defs.bzl

```diff
@@ -50,7 +50,25 @@ def _pip_import_impl(repository_ctx):
         str(repository_ctx.attr.timeout),
         "--repo-prefix",
         str(repository_ctx.attr.repo_prefix),
-    ])
+    ]
+
+    if repository_ctx.attr.extra_build_files:
+        args.extend(
+            [
+                "--extra-build-info",
+                str(
+                    struct(
+                        build_files = repository_ctx.attr.extra_build_files,
+                        build_targets = repository_ctx.attr.extra_build_targets,
+                    ).to_json(),
```

Comment:
please avoid using json, just pass it multiple times like `--extra-build-info name:file`, use `action="append"` on the python side and then split on `:`

Reply:
The character `:` is also used in the `Label` passed to the `BUILD` file. Do you have a suggestion for another character or should I just split on the 1st one and keep the rest as the `Label`?

Reply:
isn't the label translated to file paths when passed to the script?